src/harness/util.py

```diff
@@ -408,3 +409,69 @@ def filterChannelsByFrequencyRange(channels, freq_range):
       and
       channel['frequencyRange']['highFrequency'] <= freq_range['highFrequency']
   ]
+
+def _orderAttributes(obj):
+    if isinstance(obj, dict):
+        return sorted((k, _orderAttributes(v)) for k, v in obj.items())
+    if isinstance(obj, list):
+        return sorted(_orderAttributes(x) for x in obj)
+    else:
+        return obj
+
+def compareDictWithUnorderedLists(first_dict, second_dict):
+  """ Deep comparison of two dictionaries
+
+  Args:
+    first_dict: first dictionary to be compared.
+    second_dict: second dictionary to be compared.
+ Returns: boolean set to true if the dictionaries are equal and false otherwise.
+  """
+  return _orderAttributes(first_dict) == _orderAttributes(second_dict)
+
+def isPolygonCCW(polygon):
+  """
+  Args:
+    polygon: the reference GeoJSON geometry of  polygon object.
+
+   Returns: boolean set to true if the coordinates are CCW oriented  and false otherwise.
+  """
+  shapely_polygon = utils.GeoJsonToShapelyGeometry(polygon);
+  print(shapely_polygon.exterior)
+  if not shapely_polygon.exterior.is_ccw:
+    return False
+  #check the holes of the polygons
+  for ring in shapely_polygon.interiors:
+    if not ring.is_ccw:
+      return False
+  return True;
+
+def areTwoPpasEqual(first_ppa, second_ppa, delta = 10):
```

Comment:
Cosmetic note: usually when specifying the default values in arguments, we dont use space: delta=10

Reply:
fixed, thank you